.github/workflows/publish.yml

```diff
@@ -26,4 +26,4 @@ jobs:
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
         run: |
           python setup.py sdist bdist_wheel
-          twine upload --skip-existing dist/*
\ No newline at end of file
+          twine upload --skip-existing dist/*

```

Comment:
vim did this 😡 should I remove it? can black enforce one of them? 🤔

Reply:
many style guides recommend an empty line at the end of a file and I also encourage this